include/boost/math/differentiation/autodiff.hpp

```diff
@@ -499,9 +498,6 @@ long lround(const fvar<RealType,Order>&);
 template<typename RealType, size_t Order>
 long long llround(const fvar<RealType,Order>&);
 
-template<typename RealType, size_t Order>
-long long lltrunc(const fvar<RealType,Order>&);
```

Comment:
No need to remove `lltrunc()`. In fact I was going to add it anyway, exactly as you had written it. It is actually listed near the bottom of https://www.boost.org/doc/libs/1_69_0/libs/math/doc/html/math_toolkit/real_concepts.html

Reply:
I'll add it back along with a test.